python/cudf/cudf/tests/test_binops.py

```diff
@@ -881,12 +880,27 @@ def gen_df():
     utils.assert_eq(expect, got)
 
 
-@pytest.mark.parametrize("func", _operators_arithmetic + _operators_comparison)
+@pytest.mark.parametrize(
+    "func",
+    [op for op in _operators_arithmetic if op not in {"rmod", "rfloordiv"}]
+    + _operators_comparison
+    + [
+        pytest.param(
+            "rmod",
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12162"
+            ),
+        ),
+        pytest.param(
+            "rfloordiv",
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12162"
+            ),
+        ),
+    ],
```

Comment:
Not changing the behaviour of the test except to loudly xfail, rather than silently skip.

Reply:
Moved this into a separate branch.